sandbox/sandbox.go

```diff
@@ -1,3 +1,5 @@
+// +build !windows
```

Comment:
@mrjana - Why not? I don't understand. Based on previous statements, at this time we do not expect Windows to be using Sandbox, which is why _unsupported is set to build on Windows (via +build !linux). This change simply addresses the compile break from introducing the interface. As I've stated, we still fully expect docker (via libnetwork) to be passthrough on Windows down to the Hyper-V virtual switch via HCS and not using sandbox. What sandbox does on Linux has no parallel on Windows - the networking stacks are so vastly different.

(BTW, at the time of submission, I did have Windows compilation working. However, revisiting, it appears that is not the case, not sure why. Will update this PR or submit a new one when that is solved - working on it pri1. However, the sentiment of this PR still remains)